sec.club/src/views/Resources/Resources.js

```diff
@@ -0,0 +1,91 @@
+import React from "react";
```

Comment:
I was initially confused by the file structure you had here and I thought you were trying to impose an MVC type of pattern but I realized that the Resources page _is_ actually a "view" since its displaying content. Wouldn't make sense for it to be a component. Let's make sure anyone else who writes a text page puts it in view as well. Nice job.